src/Symfony/Component/Form/Extension/Core/DataAccessor/PropertyPathAccessor.php

```diff
@@ -15,7 +15,7 @@
 use Symfony\Component\Form\Exception\AccessException;
 use Symfony\Component\Form\FormInterface;
 use Symfony\Component\PropertyAccess\Exception\AccessException as PropertyAccessException;
-use Symfony\Component\PropertyAccess\Exception\NoSuchIndexException;
```

Comment:
You should keep this line as this class is referenced in this file.

Reply:
Thanks for catching this! I have added the `use` statement for `NoSuchIndexException` back to the file.